src/OrleansRuntime/Silo/Silo.cs

```diff
@@ -186,7 +186,10 @@ internal Silo(string name, SiloType siloType, ClusterConfiguration config, ILoca
 
             logger.Info(ErrorCode.SiloGcSetting, "Silo starting with GC settings: ServerGC={0} GCLatencyMode={1}", GCSettings.IsServerGC, Enum.GetName(typeof(GCLatencyMode), GCSettings.LatencyMode));
             if (!GCSettings.IsServerGC || !GCSettings.LatencyMode.Equals(GCLatencyMode.Batch))
```

Comment:
Wouldn't it be better instead not to issue the warning if we are running on a single-CPU system?